src/common/src/config.rs

```diff
@@ -115,6 +118,7 @@ pub struct RwConfig {
     pub storage: StorageConfig,
 
     #[serde(default)]
+    #[derivative(Debug = "ignore")]
```

Comment:
Will this also affect the logs from meta node?

Reply:
Users are not supposed to see this section on any node, since configuring system params from toml file is only intended for testing, not for the users.